erpnext/hr/report/vehicle_expenses/vehicle_expenses.js

```diff
@@ -22,18 +22,6 @@ frappe.require("assets/erpnext/js/financial_statements.js", function() {
 						query_report.trigger_refresh();
 					});
 				}
-			},
```

Comment:
As you have removed from_date and to_date filters, there is no point of setting those filter values on change of fiscal _year.